packages/blocks/src/image-block/image-model.ts

```diff
@@ -50,8 +50,10 @@ export class ImageBlockModel
   extends BaseBlockModel<ImageBlockProps>
   implements IEdgelessElement
 {
+  gridBound!: Bound;
```

Comment:
I'd suggest that the `gridBound` also should not exist on the model, neither does the `EdgelessSelectableMixin` related methods. But currently since we've landed #4894, removing the debt of these existing methods could be postponed in future refactoring. But at least we should not add more intrusive properties like `gridBound` for the moment.